tests/accessor/local_accessor_semantics.cpp

```diff
@@ -50,8 +50,7 @@ struct storage {
 DISABLED_FOR_TEST_CASE(hipSYCL)
 ("local_accessor common reference semantics (host)", "[local_accessor]")({
 // Issue link https://github.com/intel/llvm/issues/8332
```

Comment:
Shouldn't we remove the comment with the link to DPC++ issue?

Reply:
I was not sure if we should remove it when it's also implemented for ComputeCPP

Reply:
This issue points to DPC++ problem, not ComputeCPP, so it's better to remove to avoid confusion.

Reply:
sorry for missing this, it turned out not to be needed anymore.